src/components/dashboard/settings/AccountDeletionForm.tsx

```
@@ -0,0 +1,215 @@
+                  onClick={() => {
+                    setShowForm(false);
+                    form.reset();
+                    setMessage(null);
+                  }}
+                  className="border-gray-300 text-gray-700 hover:bg-gray-50 focus:ring-gray-500 inline-flex justify-center rounded-md border bg-white px-4 py-2 text-sm font-medium shadow-sm focus:outline-none focus:ring-2 focus:ring-offset-2"
+                >
+                  Cancel
+                </button>
+                <form.Subscribe
+                  selector={(state) => [state.canSubmit, state.isSubmitting]}
+                  children={([canSubmit, isSubmitting]) => (
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)